frontend/packages/ux-editor/src/containers/DesignView/ReceiptContent/ReceiptContent.tsx

```diff
@@ -83,7 +83,7 @@ export const ReceiptContent = ({
   }
   return (
     <div className={classes.button}>
-      <Button variant='tertiary' onClick={onClickAddPage} className={classes.button} size='small'>
+      <Button variant='tertiary' onClick={onClickAddPage} size='small'>
```

Comment:
Should we simplify this by removing the div and add the` classes.button` directly on the <Button> element itself? 😊

Reply:
That wont work. 
With your suggestion: 
<img width="323" alt="Screenshot 2023-11-29 at 09 32 10" src="https://github.com/Altinn/altinn-studio/assets/133344438/4b87825d-7b89-4277-a5e3-4491e1a68051">

With the current code: 
<img width="322" alt="Screenshot 2023-11-29 at 09 33 52" src="https://github.com/Altinn/altinn-studio/assets/133344438/6a162cab-5709-4a4e-9eee-8e73f56e7eef">